src/CMakeLists.txt

```diff
@@ -105,6 +105,8 @@ if (GLFW_BUILD_WAYLAND)
     generate_wayland_protocol("fractional-scale-v1.xml")
     generate_wayland_protocol("xdg-activation-v1.xml")
     generate_wayland_protocol("xdg-decoration-unstable-v1.xml")
+    generate_wayland_protocol("tablet-unstable-v2.xml")
```

Comment:
You could just declare a dummy `zwp_tablet_tool_v2_interface` object somewhere to avoid bringing in tablet-unstable-v2 for this, since we won't even be using it.